torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -113,8 +113,7 @@ def _get_dt_pg(dt: DTensor) -> c10d.ProcessGroup:
     mesh = dt.device_mesh
     assert mesh.ndim == 1, "Only 1D DeviceMeshes currently handled"
     dim_groups = mesh.get_group()
-    assert isinstance(dim_groups, list)
-    return dim_groups[0]
+    return dim_groups
```

Comment:
nit: its not `groups` anymore, maybe just say `return mesh.get_group()` instead?